srml/contract/src/tests.rs

```diff
@@ -900,3 +899,45 @@ fn removals(trigger_call: impl Fn() -> bool) {
 		}
 	);
 }
+
+const CODE_DEFAULT_RENT: &str = r#"
+(module
+	(import "env" "memory" (memory 1 1))
+
+	(func (export "call"))
+	(func (export "deploy"))
+)
+"#;
+const HASH_DEFAULT_RENT: [u8; 32] = hex!("4318bf9add830725edf0ff3c5f064a3aee455cb015ac560be9739deae4f700c7"); // todo
+
+#[test]
+fn default_rent_allowance_on_create() {
+	let wasm = wabt::wat2wasm(CODE_DEFAULT_RENT).unwrap();
```

Comment:
Hm, this would work but I wonder if it would be more reliable to just query `rent_allowance` within the `deploy` function and compare it with the hardcoded value?

Reply:
Do you mean calling `ext_rent_allowance` inside `deploy` in the wabt?

Reply:
yeah. What do you think?